### DISCUSSION TITLE
Filament plugin development, Tailwind classes not working

### DISCUSSION THREAD

**Discussion by kvostyc** [filamentphp/filament#8362] [Help]
Hey,

Im developing a filament plugin, i created som custom Livewire components in my package workdir.

I set a symlink for my package in composer.json so its under /vendor/ directory.

But for some reason not all Tailwind classes are working in my Filament page view, but funny is that under my package/views/ tailwind is working properly

**tailwind.config.js**
````const colors = require('tailwindcss/colors');

module.exports = {
    content: [
        './resources/**/*.blade.php',
        './resources/**/*.js',
        './vendor/filament/**/*.blade.php',
        './vendor/kvostyc/**/*.blade.php',
    ],
    theme: {
        extend: {
            colors: {
                colors,
                danger: colors.rose,
                primary: colors.red,
                success: colors.green,
                warning: colors.yellow,
                green: colors.green,
            },
        },
    },
    plugins: [
        require('@tailwindcss/forms'),
        require('@tailwindcss/typography'),
    ],
}
````
**vite.config.js**

````
import { defineConfig } from 'vite'
import laravel, { refreshPaths } from 'laravel-vite-plugin'

export default defineConfig({
    plugins: [
        laravel({
            input: ['resources/sass/app.scss', 'resources/js/app.js'],
            refresh: [...refreshPaths, 'app/Http/Livewire/**'],
        }),
    ],
})
````

**Comment by danharrin** [comment]
Are you using a custom theme?

**Comment by kvostyc** [comment]
> Are you using a custom theme?

No nothing custom, just created ServiceProvider where i loading two Resources that i created, thats working perfectly.. Just tailwind ignoring some classes in my views :/

**Reply by danharrin** [reply]
Tailwind needs to be set up in the app and not the package, as a custom theme

**Reply by bilogic** [reply]
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

![image](https://github.com/filamentphp/filament/assets/946010/117abd9e-51be-4715-b443-933fcda5f76f)

I'm facing the same issue too and followed the steps above, but my component still does not have tailwind styling even though there is definitely something in `theme.css` and I'm certain it has been included in my filament page.

**Comment by sheeehy** [comment]
Make sure that the paths you have provided in tailwind.config.js are correct and can be accessed. Your purge configuration looks to point to the ./vendor/filament and ./vendor/kvostyc directories. If you've made a symlink, verify that this symlink is working as expected.

Also, Vite might have trouble resolving symlinked directories. You might need to ensure Vite is aware of them. Check if the symlink is correctly set up, and make sure the paths in both vite.config.js and tailwind.config.js are in sync with the symlink paths.
